proposals/0048-precompile-for-secp256r1-sigverify.md

```diff
@@ -12,6 +12,10 @@ created: 2023-05-14
 feature: (fill in with feature tracking issues once accepted)
 ---
 
+
+## [DEPRECATED IN FAVOR OF SIMD-0075]
```

Comment:
(Chiming in because I also have a `supersedes` simd in progress.) We can ask how foundation wants to handle, but it doesn't make sense to me to have a document marked as superseding something that doesn't exist. I think you should leave the original simd-0048. @jacobcreech , thoughts?

Reply:
Good point. Let see what the verdict is 👍

Reply:
Reverted the commit removing the SIMD and also marked the status as withdrawn
👍

Reply:
Yes, this is the right approach. We should supersede the previous SIMD, withdraw it as it was never actually implemented, but not remove it.

Reply:
Makes sense 👍 Everything should be correct now ✅